test/helpers/helpers.test.js

```diff
@@ -98,12 +98,12 @@ describe('the getAssets function', () => {
         content: 'mockHTMLcontent',
       },
       {
-        name: 'index.html',
+        name: '/',
```

Comment:
Why did the names change? Just curious what effect this has.

Reply:
Setting the `name` sets the `friendlyName` property of the asset when it is queried from the helper library. Setting this explicitly allows us to later map existing assets to new assets being deployed in an "override" scenario.